Assets/HoloToolkit/Sharing/Tests/Scripts/ImportExportAnchorManager.cs

```diff
@@ -701,13 +693,14 @@ private void AnchorStoreReady(WorldAnchorStore store)
         /// </summary>
         private void CreateAnchorLocally()
         {
-            if (thisAnchor.isLocated)
+            WorldAnchor anchor = this.EnsureComponent<WorldAnchor>();
```

Comment:
I don't think we need to be calling `EnsureComponent` (and by extension `GetComponent`) every frame until `anchor.isLocated` returns `true`. I'd like to see a sort of hybrid between the current code and the version you have here:  

```cs
if (thisAnchor == null)
{
    thisAnchor = this.EnsureComponent<WorldAnchor>();
}
if (thisAnchor.isLocated)
{
    ...
}
```

That way we are still initializing the `WorldAnchor` component on-demand, but we only wind up calling `EnsureComponent` once.